src/js/components/Button/Button.js

```diff
@@ -475,7 +477,7 @@ const Button = forwardRef(
           }}
           onMouseOver={onMouseOverButton}
           onMouseOut={onMouseOutButton}
-          pad={!plain}
+          pad={pad || !plain}
```

Comment:
Why do we need the `!plain`, given that the Styled button code always checks the `plain` property?